language_service/src/state/tests.rs

```diff
@@ -627,6 +629,189 @@ fn close_notebook_clears_errors() {
     );
 }
 
+#[allow(clippy::too_many_lines)]
+#[tokio::test]
+async fn update_doc_updates_project() {
+    let received_errors = RefCell::new(Vec::new());
+    let mut updater = new_updater(&received_errors);
+
+    updater
+        .update_document(
+            "other_file.qs",
+            1,
+            "namespace Foo { @EntryPoint() operation Main() : Unit {} }",
+        )
+        .await;
+    updater
+        .update_document(
+            "this_file.qs",
+            1,
+            "namespace Foo {/* we should see this comment in the source*/ }",
+        )
+        .await;
+
+    check_errors_and_compilation(
+        &updater,
+        &received_errors,
+        &expect![[r#"
+            {
+                "other_file.qs": OpenDocument {
+                    version: 1,
+                    compilation: "./qsharp.json",
+                    latest_str_content: "namespace Foo { @EntryPoint() operation Main() : Unit {} }",
+                },
+                "this_file.qs": OpenDocument {
+                    version: 1,
+                    compilation: "./qsharp.json",
+                    latest_str_content: "namespace Foo {/* we should see this comment in the source*/ }",
+                },
+            }
+        "#]],
+        &expect![[r#"
+            SourceMap {
+                sources: [
+                    Source {
+                        name: "other_file.qs",
+                        contents: "namespace Foo { @EntryPoint() operation Main() : Unit {} }",
+                        offset: 0,
+                    },
+                    Source {
+                        name: "this_file.qs",
+                        contents: "namespace Foo {/* we should see this comment in the source*/ }",
```

Comment:
I don't know if it's intentional in the test, but looks like you're getting a parse error about the `/* */` comment syntax which Q# doesn't like